src/portal/src/app/interrogation-services/interrogation-services.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from "@angular/router";
+
+@Component({
+  selector: 'app-interrogation-services',
+  templateUrl: './interrogation-services.component.html',
+  styleUrls: ['./interrogation-services.component.scss']
+})
+export class InterrogationServicesComponent implements OnInit {
+
+  constructor(
+      private router: Router,
+      private route: ActivatedRoute,
+  ) {}
+
+  ngOnInit() {
+    // check scanner tab by default
+    if (!this.route.children.length) {
+      this.router.navigate(['scanners'], {relativeTo: this.route});
```

Comment:
?

Reply:
Removed this code.   And now use "redirect" in the routing module